internal/wasm/binary/table.go

```diff
@@ -30,10 +30,19 @@ func decodeTable(r *bytes.Reader, enabledFeatures api.CoreFeatures, ret *wasm.Ta
 	if ret.Min > wasm.MaximumFunctionIndex {
 		return fmt.Errorf("table min must be at most %d", wasm.MaximumFunctionIndex)
 	}
+	
+	var maxValue uint32 = 8192
```

Comment:
Can we use a constant at file level to prevent having magic numbers in the code? 😅